redash/query_runner/pg.py

```diff
@@ -28,6 +29,26 @@
 }
 
 
+class PostgreSQLJSONEncoder(JSONEncoder):
+    def default(self, o):
+        if isinstance(o, Range):
+            # From: https://github.com/psycopg/psycopg2/pull/779
+            if o._bounds is None:
+                return ''
+
+            items = [
+                o._bounds[0],
+                str(o._lower),
+                ', ',
+                str(o._upper),
+                o._bounds[1]
+            ]
+
+            return ''.join(items)
```

Comment:
Why not call `str(o)`?

Reply:
@arikfr Calling `str(o)` doesn't give proper result. My proposed implementation formats range type column values the way they can be used in an SQL query.

Also there is a comment in code that references [psycopg2#779][1], this PR is still not in the master branch of psycopg2. When [psycopg2#779][1] will make it to master then we could update psycopg2 and call `str(o)` and it would give the same result.

[1]: https://github.com/psycopg/psycopg2/pull/779

Reply:
I missed the fact it's not released yet. 👍